### ISSUE TITLE
Memory stats

### ISSUE THREAD

**Issue by Th3M1k3y** [NebN/unraid-simple-monitoring-api#4]
It would be nice to also be able to see stats about memory.

Almost same layout as a disk with total, used, free, free_percent and used_percent, leaving mount out.

**Comment by NebN** [comment]
I was thinking about this too, I'll look into it.

**Comment by NebN** [comment]
Released as part of [v0.2.2](https://github.com/NebN/unraid-simple-monitoring-api/releases/tag/v0.2.2). Check for updates on Unraid Docker tab to grab the new release.

**Comment by arcoast** [comment]
I was excited to find this, but I'm not sure if it's intentional or not yet, but the release hasn't been pushed up to dockerhub yet.

![image](https://github.com/NebN/unraid-simple-monitoring-api/assets/81871508/d1fed89b-2704-4e2e-a4ff-c1db06716553)

**Comment by Th3M1k3y** [comment]
I am not sure what happened. When looking in CA it gave me the option to install, even when it was installed. Removing the docker and installing it again made it go to the new version

**Comment by arcoast** [comment]
Yeah, I don't use CA or the Unraid docker implementation and prefer to do things in docker-compose.  I think @NebN has migrated from docker hub to ghcr.io

**Comment by NebN** [comment]
Yes I migrated from DockerHub to GHCR in [v2.2.1](https://github.com/NebN/unraid-simple-monitoring-api/releases/tag/v0.2.1). 
I did not think about how how Unraid works on the other side of things, I assumed everything would update seamlessly, but unfortunately your "personal" installation of the app still uses the old registry, so reinstalling is the only way.
I will also push to DockerHub for now and put a note on the Readme about the migration. 
Sorry for any inconvenience.
